bin/ch/WScriptJsrt.cpp

```diff
@@ -1871,3 +1871,19 @@ void WScriptJsrt::PromiseContinuationCallback(JsValueRef task, void *callbackSta
     WScriptJsrt::CallbackMessage *msg = new WScriptJsrt::CallbackMessage(0, task);
     messageQueue->InsertSorted(msg);
 }
+
+void WScriptJsrt::PromiseRejectionTrackerCallback(JsValueRef promise, JsValueRef reason, bool handled)
+{
+    Assert(promise != JS_INVALID_REFERENCE);
+    Assert(reason != JS_INVALID_REFERENCE);
+    if(!handled)
+    {
+        wprintf(_u("Uncaught promise rejection\n"));
```

Comment:
Should we also print out the message to ensure that the `reason` is being set correctly?

Reply:
If I may add further commentary to this, it might make more sense to say "Promise rejected with no handler attached" rather than "Uncaught promise rejection".  We don't know whether or not it will remain uncaught at this time.

Reply:
I've updated to print the reason.

I've left it as "Uncaught promise rejection" - as at the time of notification it is uncaught - it may be handled later but it is not at the moment of notification.